decidim-budgets/lib/decidim/budgets/component.rb

```diff
@@ -89,7 +89,7 @@
     settings.attribute :vote_selected_projects_minimum, type: :integer, default: 0
     settings.attribute :vote_selected_projects_maximum, type: :integer, default: 1
     settings.attribute :comments_enabled, type: :boolean, default: true
-    settings.attribute :comments_max_length, type: :integer, required: false
+    settings.attribute :comments_max_length, type: :integer, default: 1000, required: true
```

Comment:
Required `true` is fine to make it consistent with others but I would leave the `default` out because otherwise we are forcing the value on components (rather than using the default).
```suggestion
    settings.attribute :comments_max_length, type: :integer, required: true
```